youtube_dl/extractor/brightcove.py

```diff
@@ -508,7 +508,22 @@ def _extract_urls(webpage):
                 'http://players.brightcove.net/%s/%s_%s/index.html?videoId=%s'
                 % (account_id, player_id, embed, video_id))
 
-        return entries
+# 	<video data-brightcove-video-id="5320421710001" data-account="245991542" data-player="SJWAiyYWg" data-embed="default" class="video-js" controls itemscope itemtype="http://schema.org/VideoObject">
+        for video_id, account_id, player_id, embed in re.findall(
+                r'''(?sx)
+                    <video[^>]+
+                        data-brightcove-video-id=["\'](\d+|ref:[^"\']+)["\'].*?
+                        data-account=["\'](\d+)["\'].*?
+                        data-player=["\'](\w+)["\'].*?
+                        data-embed=["\'](\w+)["\'].*?
+                    </video>
+                ''', webpage):
```

Comment:
Done. As in #11968, of course I found myself struggling to find good examples of how it was best used, and found plenty of not-so-awesome ones...